src/platform/tls_schannel.c

```diff
@@ -1445,22 +1273,17 @@ QuicTlsInitialize(
     TlsContext->TransportParams->BufferSize =
         (uint16_t)(Config->LocalTPLength - FIELD_OFFSET(SEND_GENERIC_TLS_EXTENSION, Buffer));
 
-    //
-    // Associate the existing security config with this TLS context.
-    //
-    TlsContext->SecConfig = QuicTlsSecConfigAddRef(Config->SecConfig);
-
     State->EarlyDataState = QUIC_TLS_EARLY_DATA_UNSUPPORTED; // 0-RTT not currently supported.
+    if (Config->ResumptionTicketBuffer != NULL) {
+        QUIC_FREE(Config->ResumptionTicketBuffer);
```

Comment:
should this be set NULL here to prevent double-free?

Reply:
No. The contract is that success from this function indicates ownership of the buffer.

Reply:
Does this come from externally allocated code? If so, this is not safe to blindly free, and we either need to heavily document exactly which allocator must be used for this, find a way where we don't take ownership, or add a way to get a properly allocated buffer back.

Reply:
No. This is internal to internal code.